Dockerfile

```diff
@@ -1,18 +1,26 @@
-FROM golang:1.15 as bd
-WORKDIR /github.com/layer5io/meshery-traefik-mesh
-ADD . .
-RUN GOPROXY=https://proxy.golang.org GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-traefik-mesh .
-RUN find . -name "*.go" -type f -delete
+FROM golang:1.13 as builder
```

Comment:
1.15?

Reply:
Thanks for pointing out @mgfeller. Changing... :smile: